cmd/lsf/lsf_test.go

```diff
@@ -141,7 +141,7 @@ file3
 	items, _ := list.DirSorted(context.Background(), f, true, "")
 	var expectedOutput string
 	for _, item := range items {
-		expectedOutput += item.ModTime(context.Background()).Format("2006-01-02 15:04:05") + "\n"
+		expectedOutput += item.ModTime(context.Background()).Format("2006-01-02 15:04:05.000000000") + "\n"
```

Comment:
This is a potentially backwards incompatible change.

I think `lsf` attempts to show the correct number of digits for the Precision in the Fs.

Reply:
I don't think it does, unless there's an option I'm missing:

```
rclone lsf a --format stp --separator ' '
5 2023-11-17 01:33:40 hi copy.txt
5 2023-11-17 01:30:58 hi.txt
```
vs. `lsl`:
```
rclone lsl a                            
        5 2023-11-17 01:33:40.782855000 hi copy.txt
        5 2023-11-17 01:30:58.202211000 hi.txt
```

You are right that it could potentially break things if users are scripting with the output and expecting a certain precision (although I'd have guessed they would use `lsjson` instead.)

Reply:
I didn't change this, but I can if you think it's best.

Alternately I'm happy to implement what you described (variable precision per Fs), although that would still be a potentially backwards-incompatible change.

Reply:
This is the code that I was thinking of

https://github.com/rclone/rclone/blob/2f5685b40562cf764da673543245ddb13c049a4a/fs/operations/lsjson.go#L50

Maybe this is only used for `lsjson`?

I think we should try not to change the output of `lsf` unless it is guarded with an option.

Reply:
Got it. Yeah, looks like this only used by `lsjson`, as `lsf` does not use RFC3339. I have guarded it with a flag in the new version, so that the default is unchanged and nothing should break.